docs/fr/docs/tutorial/background-tasks.md

```diff
@@ -0,0 +1,94 @@
+# Tâches d'arrière-plan
+
+Vous pouvez définir des tâches d'arrière-plan qui seront exécutées après avoir retourné une réponse.
+
+Ceci est utile pour les opérations qui doivent avoir lieu après une requête, mais où le client n'a pas réellement besoin d'attendre que l'opération soit terminée pour recevoir une réponse.
+
+Cela comprend, par exemple :
+
+* Les notifications par email envoyées après l'exécution d'une action :
+    * Étant donné que se connecter à un serveur et envoyer un email a tendance à être «lent» (plusieurs secondes), vous pouvez retourner la réponse directement et envoyer la notification en arrière-plan.
+* Traiter des données :
+    * Par exemple, si vous recevez un fichier qui doit passer par un traitement lent, vous pouvez retourner une réponse «Accepted» (HTTP 202) puis faire le traitement en arrière-plan. 
+
+
+## Utiliser `BackgroundTasks`
+
+Pour commencer, importez `BackgroundTasks` et définissez un paramètre dans votre *fonction de chemin* avec `BackgroundTasks` comme type déclaré.
+
+```Python hl_lines="1  13"
+{!../../../docs_src/background_tasks/tutorial001.py!}
+```
+
+**FastAPI** créera l'objet de type `BackgroundTasks` pour vous et le passera comme paramètre.
+
+## Créer une fonction de tâche
+
+Une fonction à exécuter comme tâche d'arrière-plan est juste une fonction standard qui peut recevoir des paramètres.
+
+Elle peut être une fonction asynchrone (`async def`) ou une fonction normale (`def`), **FastAPI** saura la gérer correctement.
+
+Dans cet exemple, la fonction de tâche écrira dans un fichier (afin de simuler un envoi d'email).
+
+L'opération d'écriture n'utilisant ni `async` ni `await`, on définit la fonction avec un `def` normal.
+
+```Python hl_lines="6-9"
+{!../../../docs_src/background_tasks/tutorial001.py!}
+```
+
+## Ajouter une tâche d'arrière-plan
+
+Dans votre *fonction de chemin*, passer votre fonction de tâche à l'objet de type `BackgroundTasks` (`background_tasks` ici) grâce à la méthode `.add_task()` :
+
+
+```Python hl_lines="14"
+{!../../../docs_src/background_tasks/tutorial001.py!}
+```
+
+`.add_task()` reçoit comme arguments :
+
+* Une fonction de tâche à exécuter en arrière-plan (`write_notification`).
+* Les arguments positionnels à passer à la fonction de tâche dans l'ordre (`email`).
+* Les arguments nommés à passer à la fonction de tâche (`message="some notification"`).
+
+## Injection de dépendances
+
+Utiliser `BackgroundTasks` fonctionne aussi avec le système d'injection de dépendances, vous pouvez déclarer un paramètre de type `BackgroundTasks` à différents niveaux : dans une *fonction de chemin*, dans une dépendance, dans une sous-dépendance...
```

Comment:
Shouldn't we make two sentences of this one: Utiliser [...] dépendances. Vous pouvez déclarer...